internal/generate/collector/collect.go

```diff
@@ -29,6 +29,7 @@ import (
 	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
Also, the pr is missing : Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))

Could you please add?

Reply:
Done. Added the changelog fragment as suggested. Thanks.